src/ergw_gtp_c_socket.erl

```diff
@@ -406,13 +412,15 @@ handle_socket_error(Error, IP, _Port, _State) ->
     ok.
 
 handle_err_input(Socket, State) ->
-    case socket:recvmsg(Socket, [errqueue], nowait) of
+    case socket:recvmsg(Socket, [errqueue], 0) of
 	{ok, #{addr := #{addr := IP, port := Port}, ctrl := Ctrl}} ->
 	    lists:foreach(handle_socket_error(_, IP, Port, State), Ctrl),
 	    ok;
 
-	{select, SelectInfo} ->
-	    socket:cancel(Socket, SelectInfo);
+	{error, timeout} ->
+	    ok;
+	{error, ewouldblock} ->
```

Comment:
Do we need log these errors?

Reply:
no, they are expected and actually mean the same.